parts/k8s/windowsazurecnifunc.ps1

```diff
@@ -93,7 +101,25 @@ Set-AzureCNIConfig
         $configJson.plugins.AdditionalArgs[0].Value.ExceptionList = $exceptionAddresses
     }
 
-    $configJson.plugins.AdditionalArgs[1].Value.DestinationPrefix  = $KubeServiceCIDR
+    if ($IsDualStackEnabled){
+        $configJson.plugins[0]|Add-Member -Name "ipv6Mode" -Value "ipv6nat" -MemberType NoteProperty
+        $serviceCidr = $KubeServiceCIDR -split ","
+        $configJson.plugins[0].AdditionalArgs[1].Value.DestinationPrefix = $serviceCidr[0]
+        $valueObj = [PSCustomObject]@{
+            Type = 'ROUTE'
+            DestinationPrefix = $serviceCidr[1]

```

Comment:
What about the case where cluster is dual stack, but services are just single stack? The user could just have a single family CIDR for service.

Reply:
so if user didn't specify ipv6 service cidr, won't aks-e assign default service v6 cidr? May be im interpreting your ask wrongly

Reply:
aks-e default v6 CIDR only for `ClusterCIDR`. Having v4 and v6 CIDR for services isn't mandatory.

Reply:
i believe we should add default for v6 service cidr also here if user opted for dualstack. 
https://github.com/Azure/aks-engine/blob/24e9029e83d7349dcc9d1fab5e33ec0f3729e6e9/pkg/api/defaults.go#L278

Reply:
@tamilmani1989 Having dual stack CIDRs for services is not mandatory unlike `ClusterCIDRs`. The user can just define single stack v4 or v6 service CIDR in dual stack cluster. Thats the reason for not appending v6 service CIDR unless the user explicitly requests it.

Reply:
@aramase Yes you are right. Its not mandatory to create ipv6 svc but  if user decided want to have in future he has to recreate cluster. What's the user going to lose if we assign default svc cidr if its not explicitly specified?